### PR TITLE
JAMES-3946 Add a DropLists JPA backend

### PR DESCRIPTION
**Pull request by Maxxx873**

### CODE REVIEW

File: server/data/data-jpa/pom.xml [hunk 1]
```diff
@@ -33,7 +33,7 @@
     <name>Apache James :: Server :: Data :: JPA Persistence</name>
 
     <properties>
-        <target.jdk>11</target.jdk>
+        <target.jdk>21</target.jdk>
```
Comment (chibenwa) [inline server/data/data-jpa/pom.xml:36]: Is this even needed to get this property defined here?

We likely should just inherit this from parent project...

**Review by chibenwa — APPROVED** [review]
Looks overall good to me!

File: server/data/data-jpa/pom.xml (hunk 1 above)
Comment (Maxxx873) [reply server/data/data-jpa/pom.xml:36]: Yes indeed, I'll try to inherit this

**Review by quantranhong1999 — APPROVED** [review]
Thank you!

**Review by chibenwa — APPROVED** [review]

**Pull request merged**